Checking the Genuineness of your Skywallet.md

```diff
@@ -0,0 +1,76 @@
+# Checking the Genuineness of your Skywallet
+
+## Checking your Skywallet Shipment Package for Tampering
+
+Once you receive the Skywallet shipment, please check for any physical tampering or damage of any sorts to the package.
+
+Tampering or damage can be anything from a small tear on the package to a package that is torn open.
+
+**Please do not accept a shipment which seems to have tampered in any such ways.**
```

Comment:
Please change to `**Do not accept a shipment which seems to be damaged in any such way.**`